binary_transparency/firmware/cmd/flash_tool/flash_tool.go

```diff
@@ -145,7 +145,21 @@ func verifyUpdate(up api.UpdatePackage) error {
 	return nil
 }
 
+//checkConsistency checks that an update package is consistent with existing device package
 func checkConsistency(c *client.ReadonlyClient, up api.UpdatePackage, dev devices.Device) error {
-	// TODO(al): implement this
+	dc, err := dev.DeviceCheckpoint()
+	if err != nil {
+		return fmt.Errorf("failed to fetch the device checkpoint: %q", err)
```

Comment:
can you change the `%q` to `%w` to wrap the errors?

Reply:
ok, agreed will do!